app/queues/certificate/dns-cleaner-worker.server.ts

```diff
@@ -5,6 +5,7 @@ import logger from '~/lib/logger.server';
 export interface DnsCleanerData {
   rootDomain: string;
   username: string;
+  certificateId: number;
```

Comment:
Same here, use Prisma type.

Question: you are defining the same type over and over again.  Can we abstract it to a separate file and re-use across all these jobs which share the same signature?

Reply:
yes, this was my plan for whan I do the refactor

Reply:
Any preference, how to name that file?